README.md

```diff
@@ -201,7 +201,7 @@ The password for the `.pfx` file is `password`.
 
 ### Usage with Sentry
 
-To develop Relay with an existing Sentry devserver, Sentry onpremise
+To develop Relay with an existing Sentry devserver, Sentry self-hosted
```

Comment:
@chadwhitacre would it read better if this read "self-hosted Sentry installation"?

Reply:
Agreed! Done in cf7077011cb7cb610442adcdc9bd3bab7fe5509a. ☺️